pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Update your `pnpm` and `install` again. Downgrading the `lock` file should not happen.